doc/source/setup-helm.rst

```diff
@@ -58,6 +64,22 @@ cluster. At the terminal, enter:
 
 This command only needs to run once per Kubernetes cluster.
 
+.. note::
+
+   The local and remote version of ``helm`` must be the same in order to
+   ensure they can talk to each other. If you wish to run ``helm`` commands
+   from a *new* computer than the one used to run the commands above, you
```

Comment:
How about *different* instead of *new*? Or a second computer.

Reply:
I agree